### ISSUE TITLE
Position z is increasing

### ISSUE THREAD

**Issue by YapZhenYan** [ShuoYangRobotics/Cerberus#14]
I have plotted the campus bagfile as mentioned. I managed to get the shape when plotting x and y.
![pos_xy](https://github.com/ShuoYangRobotics/Cerberus/assets/122852792/b839b66e-1423-4831-8721-c7565f1edef2)

However, when I plot the z position, it is always increasing.
![pos_z](https://github.com/ShuoYangRobotics/Cerberus/assets/122852792/db2ac360-bbcb-43a8-b771-671d6c3aab1a)

**Comment by ShuoYangRobotics** [comment]
Yes, position Z is not fine-tuned and hard to be estimated correctly. Since we have other height estimation method (barometer), we do not use Cerberus for Z height estimation

**Comment by YapZhenYan** [comment]
Thank you for the information. 
1) Do you have any suggestions on how to improve the z-height estimation for Cerberus? 
2) Which file is the z position parameters being used for the state estimation?
3) How is the x and y estimation tuned in Cerberus?
